src/test/codegen/nounwind-extern.rs

```diff
@@ -1,6 +0,0 @@
-// compile-flags: -O
-
-#![crate_type = "lib"]
-
-// CHECK: Function Attrs: norecurse nounwind
-pub extern fn foo() {}
```

Comment:
This test was pretty useless: it always enabled optimizations, so even if we did not emit `nounwind` for this function, LLVM would add it. That's why we agreed in https://github.com/rust-lang/rust/pull/65020 that it could be removed.